packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeModal/StakingInfoCards/EstimatedGains.tsx

```diff
@@ -74,7 +74,15 @@ export const EstimatedGains = () => {
                     id="TR_STAKING_YOUR_EARNINGS"
                     values={{
                         a: chunks => (
-                            <TrezorLink href={HELP_CENTER_ETH_STAKING}>{chunks}</TrezorLink>
+                            <TrezorLink
```

Comment:
As far as I know, we have EstimatedGains for Solana.  Refactored component a bit to avoid this confusion https://github.com/trezor/trezor-suite/pull/16027/commits/0c3a403434ad3f3f1c13ccfde96f287d679b6557